### DISCUSSION TITLE
How to unhide the inital output in kitten ssh?

### DISCUSSION THREAD

**Discussion by siddhpant** [kovidgoyal/kitty#7465] [Q&A] [answered]
The ssh kitten hides the initial "identity added" or  "last login" output. How to unhide it?

**Comment by kovidgoyal** [comment, marked as answer]
You cannot. The ssh kitten works by running ssh with a script which then
launches your shell. When run like this ssh doesnt print that message.

**Reply by siddhpant** [reply]
Thanks.

Edit if someone stumbles here: Moved to using `SetEnv TERM=xterm-256color` instead of kitten which allows it.
